com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,62 @@
+app-id: com.adamcake.Bolt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} pillow --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/0f/8b/2ebaf9adcf4260c00f842154865f8730cf745906aa5dd499141fb6063e26/Pillow-10.0.0.tar.gz
+        sha256: 9c82b5b3e043c7af0d95792d0d20ccf68f61a1fec6b3530e718b688422727396
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - shared-modules/gtk2/gtk2.json
+  - name: openssl
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app --openssldir=/app/ssl shared zlib
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/openssl/openssl.git
+        tag: OpenSSL_1_1_1v
```

Comment:
openssl 1.1 compatibility is supposed to be in the runtime. doesn't that work?

Reply:
I don't think I've ever tried it actually. One of the games launched by Bolt depends on openssl 1.1, and that game's own flathub entry installs it from openssl.org: https://github.com/flathub/com.jagex.RuneScape/blob/master/com.jagex.RuneScape.json#L26 so I just assumed it was necessary. Give me a few minutes and I'll see what happens without it.

Reply:
Without the openssl source I get this when trying to launch the mentioned game: `error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory`